tox.ini

```diff
@@ -81,8 +93,11 @@ extras =
     docs
 
 [testenv:pur]
+deps =
+    pur==5.2.1
 commands=
-    pur -r requirements.txt
+    pur -r lint-requirements.txt
+    pur -r test-requirements.txt
```

Comment:
These commands fail now that `lint-requirements.txt` and `test-requirements.txt` no longer exist.

Reply:
Oh.. I forgot to remove this env. Long-term we'll just enable pyup or dependabot, it's not a human job.